mlir/lib/Dialect/NVGPU/IR/NVGPUDialect.cpp

```diff
@@ -362,7 +362,8 @@ std::optional<InFlightDiagnostic> verifyTmaDescriptorWithMemref(
                              << kMaxTMADimension << " but it is " << dim;
     }
   }
-  if (descMemref.getRank() > 1) {
+  if (descMemref.getRank() > 1 &&
+      descType.getSwizzle() != TensorMapSwizzleKind::SWIZZLE_NONE) {
```

Comment:
Is this change intended to be part of this patch ?

Reply:
It is not :) it was another fix, and leaked here. 
I put another PR for this fix
https://github.com/llvm/llvm-project/pull/81499